packages/providers/src/provider.ts

```diff
@@ -206,8 +206,22 @@ export default class Provider {
     /** GraphQL endpoint of the Fuel node */
     public url: string
   ) {
+    this.operations = this.createOperations(url);
```

Comment:
DRY variable assignment via method reuse.

```suggestion
    this.connect(url)
```

Reply:
This will trigger the TypeScript check as the operations are not assigned on the constructor;

<img width="398" alt="Screenshot 2022-12-22 at 10 27 57" src="https://user-images.githubusercontent.com/3941923/209202560-586c98aa-28fb-4aa5-9c21-70bdcffb249b.png">